### ISSUE TITLE
Private Link Blob in Cloud Witness

### ISSUE THREAD

**Issue by grayjeremy** [MicrosoftDocs/windowsserverdocs#4452]
I am not seeing an explicit callout for using a private link enabled storage account.  Typing in the account name and assuming the "blob.core.windows.net" will circumvent the ability to use a private storage account. 

I am assuming the powershell command Set-ClusterQuorum with the option Endpoint as "privatelink.blob.core.windows.net" will allow the use of a privateLink storage account.  Can we create a section that addresses this near the "to create an azure storage account" section and provide the powershell example?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7e6da99e-a148-b5ea-224d-8a4c48604945
* Version Independent ID: a6a90a1f-870c-3501-ce6e-fbf52f802dfb
* Content: [Deploy a cloud witness for a Failover Cluster](https://docs.microsoft.com/en-us/windows-server/failover-clustering/deploy-cloud-witness#configure-cloud-witness-as-a-quorum-witness-for-your-cluster)
* Content Source: [WindowsServerDocs/failover-clustering/deploy-cloud-witness.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/failover-clustering/deploy-cloud-witness.md)
* Product: **windows-server**
* Technology: **storage-failover-clustering**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

**Comment by illfated** [comment]
Sure, that sounds reasonable to me. I am not affiliated with Microsoft in any way, but I am happy to provide a pull request for the required material, once I know what that content should look like. I must admit that this topic is unfamiliar to me and that I only know enough Windows Server management to understand the concept superficially. Therefore, it would be preferable if someone familiar with this topic could provide the initial phrasing for factual correctness.

At first, I thought that maybe the PowerShell cmdlet might belong under [**Configuring Cloud Witness using PowerShell**](https://docs.microsoft.com/en-us/windows-server/failover-clustering/deploy-cloud-witness#configuring-cloud-witness-using-powershell), but perhaps your requested topic is a special case and needs to be grouped as you said.

Looking forward to your reply.

/Trond B. K. a.k.a. @illfated

**Comment by grayjeremy** [comment]
I think the example should likely be in the "Configuring Cloud Witness using PowerShell" section but it needs a link/callout in "To configure Cloud Witness as a Quorum Witness" on Step 5 (describing figure 9)

**Comment by illfated** [comment]
Good point. I agree. I'll see what I can do to figure that out. In the meantime, feel free to post any suggestions you might have for the wording, if you can think of any.

**Comment by AamirulHassan** [comment]
Hi,
I have read that Cloud Witness must require the internet access on servers where it is connected and cluster role installed. but in following scenario,
- 2 Azure IaaS servers for SQL Always On, 
- Cluster quorum is cloud witness
- Both servers are behind the Azure internal load balancer  (internet is not working on servers)
But in the cluster resources Cloud Witness is showing green and everything is working apparently. 

Will Azure VMs in same subscription, resource group, same Vnet, doesnt require the internet to connect with cloud witness? or it is connected through service endpoints if enabled in the same Vnet.

Regards

**Comment by grayjeremy** [comment]
@AamirulHassan, I think you are asking a separate question, I'd try creating another issue.

**Comment by AamirulHassan** [comment]
Hi Gray,
Thanks for notice, can you please create another ticket or just guide me how to, because i am new here.

**Comment by illfated** [comment]
@AamirulHassan : Is your ticket about the same page as Jeremy's page, https://docs.microsoft.com/en-us/windows-server/failover-clustering/deploy-cloud-witness#configure-cloud-witness-as-a-quorum-witness-for-your-cluster, or is your issue related to a different page?

When you locate the page where you want to address the documentation, or suggest changes, go to the bottom of the page and click [:octocat:  **This page**]

![image](https://user-images.githubusercontent.com/38162891/84780824-57e22a00-afe6-11ea-8e19-e684a700e6ed.png)


The issue creation button for this specific page is as follows:
***
https://github.com/MicrosoftDocs/windowsserverdocs/issues/new?title=&body=%0A%0A%5BEnter%20feedback%20here%5D%0A%0A%0A---%0A%23%23%23%23%20Document%20Details%0A%0A%E2%9A%A0%20*Do%20not%20edit%20this%20section.%20It%20is%20required%20for%20docs.microsoft.com%20%E2%9E%9F%20GitHub%20issue%20linking.*%0A%0A*%20ID%3A%207e6da99e-a148-b5ea-224d-8a4c48604945%0A*%20Version%20Independent%20ID%3A%20a6a90a1f-870c-3501-ce6e-fbf52f802dfb%0A*%20Content%3A%20%5BDeploy%20a%20cloud%20witness%20for%20a%20Failover%20Cluster%5D(https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fwindows-server%2Ffailover-clustering%2Fdeploy-cloud-witness%23configure-cloud-witness-as-a-quorum-witness-for-your-cluster)%0A*%20Content%20Source%3A%20%5BWindowsServerDocs%2Ffailover-clustering%2Fdeploy-cloud-witness.md%5D(https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fwindowsserverdocs%2Fblob%2Fmaster%2FWindowsServerDocs%2Ffailover-clustering%2Fdeploy-cloud-witness.md)%0A*%20Product%3A%20**windows-server**%0A*%20Technology%3A%20**storage-failover-clustering**%0A*%20GitHub%20Login%3A%20%40JasonGerend%0A*%20Microsoft%20Alias%3A%20**jgerend**
***

**Comment by angabe89** [comment]
Hi

I am implementing a Cluster and we need to confirm if Private Endpoints are supported for Cloud Witness. Any comments about this issue?

**Comment by illfated** [comment]
Wish I could help, but this repository is not intended for regular support issues, support requests or support questions.

You may want to look elsewhere for answers, unless you have suggestions for improving one or more specific pages in the MS Docs repository.

Disclaimer: I am not affiliated with Microsoft or the Microsoft Docs teams here on GitHub.

The intention of this repository is to improve on the existing documentation by providing the users a way to report factual errors, improvement suggestions and request updates for new versions.

The MS Docs team members might not have time to answer your question, both due to the small size of the team and the big workload.

Try the following resources to get in touch with users who may be able to help you:
- TechNet forums: https://social.technet.microsoft.com/Forums/en-US/home
- MSDN: https://social.msdn.microsoft.com/Forums/en-US/home
- Microsoft Community: https://answers.microsoft.com/

**Comment by angabe89** [comment]
Hi 

I understand that this is not a support channel. I just ask for a complete/ updated documentation for Cloud Witness. There are 10 requests to improve the existing documentation (including this one) about Cloud Witness deploy and none have been answered.

**Comment by illfated** [comment]
Very well, fair enough. I have previously made some efforts to look into this earlier, but I admit that I am not an expert on this topic. The MS Docs server team is quite small, so I assume it may take quite some time before we see the improvements you have requested.

**Comment by jcbrooks92** [comment]
@grayjeremy I know this is an older post but wanted to provide feedback. When using private link, the domain used to access the storage account from an application or service doesn't change. The only thing that changes is updating your DNS with an A record for accountame under the privatelink.blob.core.windows.net zone (whether in your customer DNS server or private dns zone).

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
